index.html

```diff
@@ -510,6 +519,55 @@ <h3>Security and Privacy Considerations</h3>
 
 		</section>
 
+		<section id="audio-ua-behaviour">
+			<h2>User Agent Behaviours for Audiobooks</h2>
```

Comment:
Is this section normative? I am not sure.

- if not, this must be clearly stated 
- if yes, then the terms 'must', 'recommended', etc., should be used in the standard sense and rendering throughout the section...

(Personally, I believe it is not normative, but that should be a WG decision)

Reply:
It can't be normative and also say "All user agent behaviours described in this section are intended to provide implementors with guidance, not strict requirements."